addons/website_livechat/static/tests/embed/livechat_button_patch_tests.js

```diff
@@ -0,0 +1,25 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+
+import { loadDefaultConfig, start } from "@im_livechat/../tests/embed/helper/test_utils";
+
+import { click, contains } from "@web/../tests/utils";
+
+QUnit.module("live chat button");
+
+QUnit.test("clicking on notification opens the chat", async () => {
+    await startServer();
+    await loadDefaultConfig();
+    await start({
+        mockRPC(route) {
+            if (route === "/im_livechat/init") {
+                return { rule: { action: "display_button_and_text" } };
+            }
+        },
+    });
+    await click(".o-livechat-LivechatButton-notification", {
+        text: "Have a Question? Chat with us.",
+    });
+    await contains(".o-mail-ChatWindow");
+});
```

Comment:
Ok for fix for now, but note that test is costly to maintain. No implementation of `im_livechat/init` in mock server forces test to fully mock it, which can make test pass or fail in a way that doesn't reflect whether the feature actually works or not, which leads to drastically slowing down future devs.